services/github/src/handlers/job.js

```diff
@@ -239,6 +239,17 @@ export async function jobHandler(message) {
     }
     if (!graphConfig.tasks || graphConfig.tasks.length === 0) {
       debug(`intree config for ${organization}/${repository}@${sha} compiled with zero tasks. Skipping.`);
+
+      // If triggered by a comment, add a reaction to indicate no tasks were generated
```

Comment:
That comment is wrong now

Reply:
lol .. too many things are happening at once 🙈  thanks